main.css

```diff
@@ -442,3 +449,86 @@ button:hover {
 .proj-cad {
   display: grid;
 }
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0;
+  width: 372px;
+}
+
+.bg-image {
+  background-image: url(images/shapes.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.contact-title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 56px;
+  justify-content: center;
+  padding-top: 80px;
+}
+
+.contact-text {
+  padding-left: 24px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  justify-content: center;
+  width: 320.5px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0.5px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin: 0 20px;
+}
+
+#username {
+  background: #fff;
+  height: 30px;
+  width: 327px;
+  left: 16px;
+  right: 38px;
+  margin: 5px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+#useremail {
+  background: #fff;
+  height: 30px;
+  width: 327px;
+  left: 16px;
+  right: 38px;
+  margin: 5px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+#usermessage {
+  height: 170px;
+  width: 327px;
+  left: 24px;
```

Comment:
- [ ] Please, consider the input text alignment, spaces, and size, to be much more similar as the Figma template.
This is the Figma template:
<img width="308" alt="Screen Shot 2022-01-11 at 22 41 00" src="https://user-images.githubusercontent.com/49064106/149002225-72d7c128-e7bb-4bd8-8699-9ce6e3c3dbc2.png">